src/ext/java/org/opentripplanner/ext/fares/impl/OrcaFareService.java

```diff
@@ -594,6 +636,13 @@ private boolean permitsFreeTransfers(RideType rideType) {
     };
   }
 
+  private static boolean agencyAcceptsOrca(RideType rideType) {
```

Comment:
This static method only operates on a single type and can therefore be made an instance method.

Reply:
Also applies to `permitsFreeTransfers`.